buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -8,4 +8,7 @@ object Versions {
     const val protobufJs = "6.10.2"
     const val springBootGradlePlugin = "2.3.7.RELEASE"
     const val osDetectorGradlePlugin = "1.6.2"
+    const val androidGradlePlugin = "4.1.2"
```

Comment:
There is an argument to be made that `androidMinSdk` is actually a configuration rather than a version, but decided to keep it here for simplicity. Happy to move this around.